### DISCUSSION TITLE
Music plugin (alternative versions)

### DISCUSSION THREAD

**Discussion by Danny-Vera** [runelite/runelite#18360] [Ideas]
Hello,

I'm not a developer myself, so I hope some hobbyist picks this up and wants to create my idea for a RuneLite plugin.

There's more and more versions of (Old School) Runescape music coming out, as such:
- [Battleaxes and Ballads](https://open.spotify.com/album/6zzpG0bUeTp6dAKSkqraUc?si=DatWwKbDQB6aWKoKwuuR4g)
- [Old School Runebreaks](https://open.spotify.com/album/6sKCkbyszg3qexRP0G0png?si=vkpAMnKTT7qN1NSo6RO80w)
- [The Orchestral Collection](https://open.spotify.com/album/3b9NjRHMF4NBqviA3LKj2j?si=uyt25bXLRSGSbod031x1Eg)

Some of these albums have exclusive OSRS tracks, some are mixed with RS3 tracks.

Anyway, my idea would be to have a plugin where it would replace the currently played song with an alternative version (see above) automatically. This would mean not all OSRS tracks would be replaced, but only the selected few.

I hope this sounds interesting for those developers with time on their hands.

Regards,
Danny Vera (In game name)

**Comment by YeahImRose** [comment]
Love the idea, hopefully at some point more of the music API gets exposed, because it'd be very difficult to do this with how the API is now.

**Reply by RevoSucks** [reply]
Could you elaborate more? I want the ability to use the pre-2007 version(s) of the tracks, and this seems similar, but not exactly.
